sql/core/src/main/scala/org/apache/spark/sql/UDFRegistration.scala

```diff
@@ -412,6 +419,63 @@ class UDFRegistration private[sql] (functionRegistry: FunctionRegistry) extends
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
+   * Register a Java UDF class
+   * @param name
+   * @param className
+   * @param returnType
+   */
+  def registerJava(name: String, className: String, returnType: DataType): Unit = {
+
+    try {
+      // scalastyle:off classforname
```

Comment:
This style rule is here to prevent misuse.  Is there a reason we aren't using our utility functions?